o8g/Scripts/actions.py

```diff
@@ -984,7 +984,7 @@ def nextEncounter(group, x, y, facedown, who=me):
     if x == 0 and y == 0:  #Move to default position in the staging area
         addToStagingArea(card, facedown, who)       
     else:
-        card.moveToTable(x-card.width()/2, y-card.height()/2, facedown)
+        card.moveToTable(x, y, facedown)
```

Comment:
this is actually a behavior change. not sure it matters much. The least invasive fix would be to change `card.width()` to `card.width` which I believe is an API change and do the same thing for `height()`. The previous change centers the card where the mouse is where as this puts the card's top left corner where the mouse is.